src/main/scala/firrtl/Emitter.scala

```diff
@@ -60,6 +61,8 @@ object EmitCircuitAnnotation extends HasShellOptions {
             Seq(RunFirrtlTransformAnnotation(Dependency(Btor2Emitter)), EmitCircuitAnnotation(Btor2Emitter.getClass))
           case "experimental-smt2" | "smt2" =>
             Seq(RunFirrtlTransformAnnotation(Dependency(SMTLibEmitter)), EmitCircuitAnnotation(SMTLibEmitter.getClass))
+          case "experimental-rtlil" | "rtlil" =>
```

Comment:
For now, only `experimental-rtlil`. The SMT backend went through a year of updates until we decided it was time to allow the use without the `experimental-` prefix. Essentially by forcing users to acknowledge that this is experimental we reserve the right to radically change or remove the backend in the future.